design/cstor/v1/pool_expansion_by_expanding_disk.md

```diff
@@ -0,0 +1,145 @@
+---
+oep-number: 2
+title: Pool Expansion When Underlying Disk Expanded
+authors:
+  - "@mittachaitu"
+owners:
+  - "@vishnuitta"
+  - "@kmova"
+  - "@prateekpandey14"
+  - "@sonasingh46"
+editor: "@mittachaitu"
+creation-date: 2020-03-20
+last-updated: 2020-03-20
+status: Implementable
+---
+
+# Pool Expansion When Underlying Disk Expanded
+
+## Table of Contents
+
+- [Pool Expansion When Underlying Disk Expanded](#pool-expansion-when-underlying-disk-expanded)
+	- [Table of Contents](#table-of-contents)
+	- [Summary](#summary)
+	- [Motivation](#motivation)
+		- [Goals](#goals)
+		- [Non-Goals](#non-goals)
+	- [Proposal](#proposal)
+		- [User Stories](#user-stories)
+		- [Current Implementation](#current-implementation)
+		- [Proposed Implementations](#proposed-implementations)
+		- [Steps to perform user stories](#steps-to-perform-user-stories)
+			- [Pool Expansion Steps:](#pool-expansion-steps)
+		- [Low Level Design](#low-level-design)
+			- [Work Flow](#work-flow)
+	- [Drawbacks](#drawbacks)
+	- [Alternatives](#alternatives)
+	- [Infrastructure Needed](#infrastructure-needed)
+
+## Summary
+
+- This proposal includes design details of the pool expansion when underlying disks were expanded.
+
+## Motivation
+
+There were cases where OpenEBS users expand the pool by expanding underlying disks(which were used by pool) when space on the pool got exhausted.
+
```

Comment:
Add: This design proposal is to automate the manual steps mentioned [here](https://gist.github.com/prateekpandey14/f2a30b3f246fd5b44fdfb545185f78b4)

Reply:
Added the doc link